devicetypes/smartthings/centralite-thermostat.src/centralite-thermostat.groovy

```diff
@@ -81,9 +81,11 @@ metadata {
 // parse events into attributes
 def parse(String description) {
 	log.debug "Parse description $description"
+	def result = null
 	def map = [:]
+ 	def additionalMap = [:]
 	if (description?.startsWith("read attr -")) {
-		def descMap = parseDescriptionAsMap(description)
```

Comment:
You can remove the definition of `parseDescriptionAsMap` since you're using the `zigbee` version now